README.md

```diff
@@ -188,31 +200,23 @@ Approvers ([@open-telemetry/js-approvers](https://github.com/orgs/open-telemetry
 
 *Find more about the approver role in [community repository](https://github.com/open-telemetry/community/blob/main/community-membership.md#approver).*
 
-Maintainers ([@open-telemetry/js-maintainers](https://github.com/orgs/open-telemetry/teams/javascript-maintainers)):
+#### Emeriti
 
-- [Amir Blum](https://github.com/blumamir), Aspecto
-- [Chengzhong Wu](https://github.com/legendecas), Alibaba
-- [Daniel Dyla](https://github.com/dyladan), Dynatrace
-- [Marc Pichler](https://github.com/pichlermarc), Dynatrace
-- [Rauno Viskus](https://github.com/Rauno56), Splunk
-- [Valentin Marchaud](https://github.com/vmarchaud), Open Source Contributor
+- [Bartlomiej Obecny](https://github.com/obecny), LightStep, Maintainer
+- [Daniel Khan](https://github.com/dkhan), Dynatrace, Maintainer
+- [Mayur Kale](https://github.com/mayurkale22), Google, Maintainer
+- [Rauno Viskus](https://github.com/rauno56), Maintainer
+- [Brandon Gonzalez](https://github.com/bg451), LightStep, Approver
```

Comment:
is there a specific order ?

Reply:
The idea was alphabetical in the past but it looks like the "previous approver/maintainer" section didn't follow it